content/en/docs/concepts/glossary.md

```diff
@@ -177,13 +177,13 @@ A language-specific collection of behavior invoked by an interface.
 Sometimes used to refer to a collection of [Log records](#log-record). Can be
 ambiguous since people also sometimes use [Log](#log) to refer to a single
 [Log record](#log-record). Where ambiguity is possible, use additional
-qualifiers, for example, `Log record`. See [more][log]
+qualifiers, for example, `Log record`. See [Log].
```

Comment:
Non-blocking, but I just want to make sure we want to link to the spec glossary definition here and not the [Logs](https://opentelemetry.io/docs/concepts/signals/logs/) concept page as we do for ### Metrics.

Reply:
Ah, right. There are actually other problems with this entry. I'll create a separate tracker issue for this.

Reply:
Will track via:

- #6985

Reviewers, please comment in that issue.